engine/env/spark/src/main/java/org/datacleaner/spark/SparkAnalysisRunner.java

```diff
@@ -115,9 +117,12 @@ public AnalysisResultFuture run(AnalysisJob job) {
     }
 
     private boolean isDistributable(AnalysisJob job) {
-        for (AnalyzerJob analyzerJob : job.getAnalyzerJobs()) {
-            if (!analyzerJob.getDescriptor().isDistributable()) {
-                return false;
+        for (ComponentJob componentJob : _sparkJobContext.getComponentList()) {
+            if (componentJob instanceof AnalyzerJob) {
```

Comment:
Actually _all_ components have a distributable flag, so you shouldn't filter on analyzers here.